content/asp.net/getting-started/app-services.md

```diff
@@ -6,15 +6,190 @@ Download a ZIP containing this tutorial's sample files:
 - [Initial Version] - Use this as a starting point when following along with the tutorial yourself
 - [Completed Version] - Includes the completed versions of all samples
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/53) to claim this lesson and/or view what should be included in it.
+## Introduction
 
-## First Header
+Your ASP.NET Core app will likely depend on a number of services. These services perform important functionality for the app. Almost anything your app does could be encapsulated in a service, but the most important operations to isolate are those that work with infrastructure outside the scope of your app. For instance, if your app stores data in a database, or reads data in from an online API, these operations would benefit from being encapsulated in services rather than being performed directly from the app. ASP.NET Core is built on the premise of modularity, and includes built-in functionality for registering services your app requires and making these services available to classes within your app. You register services your app will need in the `ConfigureServices` method in *Startup.cs*. Your classes that depend on these services then request them by declaring parameters in their constructors. Writing classes such that they explicitly request their dependencies is said to follow the [Explicit Dependencies Principle](http://deviq.com/explicit-dependencies-principle/), which is closely related to the [Dependency Inversion Principle](http://deviq.com/dependency-inversion-principle/).
```

Comment:
>You register services your app will need in the `ConfigureServices` method in *Startup.cs*.

We're talking about both `Configure` and `ConfigureServices` in this one...maybe a mention here so it's understood that both are found in `startup.cs`? Or is that too obvious?